devs/docs/basics.rst

```diff
@@ -132,6 +132,10 @@ This will set up the project using the pre-configured code style, code
 inspection, etc. It will also create some run/debug configurations which
 allows you to start Crate from the IDE.
 
+After setting up code style, it can be checked by running::
+
+    ./gradlew checkstyleMain
+
```

Comment:
I think this would fit into it's own section as it is unrelated to using an IDE.

Reply:
Yep, we have a dedicated `Checkstyle` section, moved it there